presto-native-execution/presto_cpp/presto_protocol/core/presto_protocol_core.yml

```diff
@@ -32,14 +32,14 @@ ExtraFields:
     std::shared_ptr<SourceLocation>: sourceLocation
 
   Aggregation:
-    FunctionHandle:      functionHandle
+    FunctionHandle: functionHandle
```

Comment:
@rschlussel : You will have to regenerate the native code protocol classes using the instructions here https://github.com/prestodb/presto/tree/master/presto-native-execution/presto_cpp/presto_protocol#presto-native-worker-protocol-code-generation.

Post that you would need more changes in C++ plan node conversions (from Presto node protocol classes -> Velox PlanNodes) as well I think.

Reply:
i did.  I ended up with some white space changes to presto_protocol_core.h that were incorrect (failed the format check), so I reverted those, but otherwise these are the only changes I had.

Reply:
Thanks for confirming @rschlussel